pages/index.html

```diff
@@ -4,9 +4,15 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Rebeca Muniz</title>
```

Comment:
It is a convention to organize an html project such that `index.html` is at the project root level, and the other pages are in the `pages` directory.